vector/src/main/res/values/strings.xml

```diff
@@ -3653,4 +3653,29 @@
         <item quantity="one">At least %1$s option is required</item>
         <item quantity="other">At least %1$s options are required</item>
     </plurals>
+    <plurals name="poll_option_vote_count">
+        <item quantity="one">%1$s vote</item>
+        <item quantity="other">%1$s votes</item>
+    </plurals>
+    <plurals name="poll_total_vote_count_before_ended_and_voted">
+        <item quantity="one">Based on %1$s vote</item>
+        <item quantity="other">Based on %1$s votes</item>
+    </plurals>
+    <plurals name="poll_total_vote_count_before_ended_and_not_voted">
+        <item quantity="zero">No votes cast</item>
+        <item quantity="one">%1$s vote cast. Vote to the see the results</item>
+        <item quantity="other">%1$s votes cast. Vote to the see the results</item>
+    </plurals>
+    <plurals name="poll_total_vote_count_after_ended">
+        <item quantity="one">Final result based on %1$s vote</item>
+        <item quantity="other">Final result based on %1$s votes</item>
```

Comment:
I prefer that we use `%1$d` instead of `%1$s` when integers will be used to build the string. To be changed at 8 places.

Reply:
Of course. Done.